package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "forms-extension",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Used by french employees to generate their holiday form",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint src lib",
+    "test": "mocha"
```

Comment:
note: you can add `"start": "cd src && jpm run"` this makes it easier to test in Firefox.

If you want to use a specific firefox you can use: `npm start -- -b /path/to/firefox`